code/lib/types/src/modules/store.ts

```diff
@@ -35,15 +35,18 @@ export type Store_ModuleImportFn = (path: Store_Path) => Store_PromiseLike<Store
 type Store_MaybePromise<T> = Promise<T> | T;
 
 export type Store_TeardownRenderToDOM = () => Store_MaybePromise<void>;
-export type Store_RenderToDOM<TFramework extends AnyFramework> = (
+// We should consider renaming this to `RenderToRootElement` or similar
+export type Store_RenderToDOM<TFramework extends AnyFramework, TRootElement = HTMLElement> = (
   context: Store_RenderContext<TFramework>,
-  element: Element
+  element: TRootElement
 ) => Store_MaybePromise<void | Store_TeardownRenderToDOM>;
 
-export type Store_WebProjectAnnotations<TFramework extends AnyFramework> =
-  ProjectAnnotations<TFramework> & {
-    renderToDOM?: Store_RenderToDOM<TFramework>;
-  };
+export type Store_WebProjectAnnotations<
```

Comment:
Now it is parameterized, we might consider moving `renderToDOM` (aka `renderToRootElement`) into the CSF project annotation type proper. 

WDYT @kasperpeulen? LMK if you want to talk about this.

Reply:
Do we have a CSF project annotations type yet?

Reply:
@shilman Yes we have. 

@tmeasday It sounds good to me, I don't know much about renderToDom, but seems like a nice one to add to CSF.

Reply:
`renderToRootElement` seems very tied to our implementation. Without more information I'd argue against moving it to CSF, which is trying to be portable

Reply:
Let's discuss @shilman. I hear you for sure, but this also feels like a loose end (literally one field on project annotations which isn't in CSF).